codalab/worker/worker_run_state.py

```diff
@@ -125,6 +125,16 @@ class RunStage(object):
 DependencyToMount = namedtuple('DependencyToMount', 'docker_path, child_path, parent_path')
 
 
+def log_bundle_transition(
+    bundle_uuid: str, previous_state: str, next_state: str, reason: str, warn: bool = False
```

Comment:
There should be a type for states that's not `str`.

Reply:
I think we are actually logging transitions of stages in this function. It's only updating `stage` of the run_bundle. Stages like `RunStage.CLEANING_UP` are defined as strs right?